pkg/service/replay/replay.go

```diff
@@ -452,6 +452,12 @@ func (r *replayer) RunTestSet(ctx context.Context, testSetID string, testRunID s
 			utils.LogError(r.logger, nil, "test result is nil")
 			break
 		}
+
+		// We need to sleep for a second to avoid mismatching of mocks during keploy testing via test-bench
+		if r.config.EnableTesting {
+			r.logger.Debug("sleeping for a second to avoid mismatching of mocks during keploy testing via test-bench")
```

Comment:
During recording, If we do it manually, there will be a significant delay (let's say in ms) but still it will be slower than an execution where it is done sequentially in a for loop. To avoid mismatch due to timestamps overlapping, I am sleeping for a second before the next simulated call.